src/proteus/dswx_hls.py

```diff
@@ -3479,7 +3479,7 @@ def _populate_dswx_metadata_processing_parameters(
 
     # Copernicus class for forest masking
     dswx_metadata_dict['FOREST_MASK_LANDCOVER_CLASSES'] = \
-        ', '.join(forest_mask_landcover_classes)
+        ','.join([str(c) for c in forest_mask_landcover_classes])
```

Comment:
For the metadata, should class 201 (evergreen) also be included in this list? Even though this class is hardcoded into the algorithm, it is one of the Copernius forest classes which is being masked.

Reply:
Per meeting with J. Jones, class `201` should not be included. (This was a misunderstanding on my part.)